internal/model/id.go

```diff
@@ -17,28 +18,23 @@ var (
 )
 
 type TMID struct {
-	Name         string
-	OptionalPath string
-	Author       string
-	Manufacturer string
-	Mpn          string
-	Version      TMVersion
+	Name    string
+	Version TMVersion
 }
 
 func NewTMID(author, manufacturer, mpn, optPath string, version TMVersion) TMID {
```

Comment:
We should add a test here, that the result name is sanitized.

Reply:
done